src/config/texts.js

```diff
@@ -422,6 +422,12 @@ export const texts = {
     configureReminder: 'Erinnerungen einstellen',
     errorOnUpdateBody: 'Beim Aktualisieren Ihrer Einstellungen ist ein Fehler aufgetreten.',
     errorOnUpdateTitle: 'Fehler',
+    exportAlertBody:
+      'Ein Download wurde gestartet.\nNachdem der Download abgeschlossen ist, können Sie die Termine durch öffnen der Datei in Ihren Kalender importieren.',
```

Comment:
- [ ] `durch öffnen` -> `durch Öffnen`

Reply:
Changed other parts of the string, but fixed this issue along with it 💪